ui/packages/consul-ui/app/templates/application.hbs

```diff
@@ -50,7 +50,9 @@ as |source|>
 {{! until we get to the dc route we don't know any permissions }}
 {{! as we don't know the dc, any inital permission based }}
 {{! redirects are in the dc.show route}}
-  {{did-insert (route-action 'replaceWith' 'dc.show'
+
+{{! 2022-04-15: Temporarily reverting the services page to the default }}
+  {{did-insert (route-action 'replaceWith' 'dc.services.index'
```

Comment:
Confirming that this reverts to the previous routing behavior, as seen in this PR: https://github.com/hashicorp/consul/commit/4b8cabce0df97c4a396c4b843dc5fe46d2c40c92#diff-f22be8f0dfc39d13e267d1dbf004a6459dee10397b048c14e97ab5df8bc44e61R50

Route used to go to `dc.services.index`.